client/src/services/sunfishapi.service.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+app.factory('SunfishSrvc', ['$http', function($http){
+    var baseUrl = '/api/siafile/';
+    var sunfishSrvc = {};
```

Comment:
Did "Service" really need to be shortened to "Srvc"? You save a grand total of THREE letters